### DISCUSSION TITLE
Tests fails after upgrade from version@0.30.0 to 0.34.3

### DISCUSSION THREAD

**Discussion by VickTor61** [vitest-dev/vitest#4037] [Q&A]
My test suites were all passing appropriately while using the vitest test runner on the c8 coverage engine but after i made the upgrade to use 0.34.3 and the v8 coverage engine, i get errors i do not understand.

**Comment by wyardley** [comment]
What's the error your getting, and is it locally, or in CI?

Just throwing up a link to:
https://github.com/vitest-dev/vitest/issues/4044

in case it's a similar error.
